src/main/scala/com/exini/dicom/streams/ParseFlow.scala

```diff
@@ -139,7 +145,7 @@ class ParseFlow private (chunkSize: Int) extends ByteStringParser[DicomPart] {
           else
             InDeflatedData(DeflatedState(state.bigEndian, nowrap = true))
         else
-          InDatasetHeader(DatasetHeaderState(0, bigEndian, explicitVR))
+          InDatasetHeader(DatasetHeaderState(0, maySwitchTs = true, bigEndian, explicitVR))
```

Comment:
By setting maySwitchTs to true here we are able to parse data that claims to be implicit VR but is really explicit VR